.github/workflows/release.yml

```diff
@@ -0,0 +1,126 @@
+        with:
+          toolchain: ${{ steps.getrustver.RUST_VER }}
+
+      - name: setup msys2 for windows
+        if: matrix.os == 'windows-latest'
+        uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          install: mingw-w64-x86_64-toolchain mingw-w64-x86_64-sqlite3 gcc make base-devel mingw-w64-x86_64-rust zip
+          msystem: mingw64
+          path-type: inherit
+
+      - name: Build sqld (windows)
+        if: matrix.os == 'windows-latest'
+        run: |
+          export CARGO_BIN="$(which cargo)"
+          cargo build --release 
+        shell: msys2 {0}
+
+      - name: Zip artifact for deployment (windows)
+        if: matrix.os == 'windows-latest'
+        #run: zip sqld-${{ github.event.inputs.tag }}-${{ matrix.os }}.zip ./target/release/sqld
+        run: zip sqld-${{ github.ref }}-${{ matrix.os }}.zip ./target/release/sqld
+        shell: msys2 {0}
+      
+      - name: install dependencies (linux)
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt install --yes \
+            bundler \
+            libpq-dev \
+            libsqlite3-dev \
+            nodejs \
+            protobuf-compiler 
+
+      - name: Build sqld (linux/macos)
+        if: matrix.os != 'windows-latest'
+        run: |
+          cargo -V
+          git status
+          git describe --tags
+          cargo build --release 
+
+      - name: Zip artifact for deployment (linux/macos)
+        if: matrix.os != 'windows-latest'
+        #run: zip sqld-${{ github.event.inputs.tag }}-${{ matrix.os }}.zip ./target/release/sqld
+        run: zip sqld-${{ github.ref }}-${{ matrix.os }}.zip ./target/release/sqld
+
+      - uses: actions/upload-artifact@v3
+        with:
+          #name: sqld-${{ github.event.inputs.tag }}-${{ matrix.os }}
+          #path: sqld-${{ github.event.inputs.tag }}-${{ matrix.os }}.zip
+          name: sqld-${{ github.ref }}-${{ matrix.os }}
+          path: sqld-${{ github.ref }}-${{ matrix.os }}.zip
+          retention-days: 1
+
+  release:
+    name: "Release"
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v3
+
+      - name: Display fetched artifacts
+        run: ls -Rha
+
+        
+      # NOTE: When providing a body and body_path at the same time, 
+      # body_path will be attempted first, then falling back on body 
+      # if the path can not be read from.
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          token: "${{ secrets.GH_TOKEN }}"
+          body: |
+            SQLd ${{ github.ref }}" 
+          body_path: RELEASE_NOTES.md
+          prerelease: false
+          tag_name: ${{ github.ref_name }}
+          generate_release_notes: true
+          fail_on_unmatched_files: false
+          files: |
+             sqld-${{ github.ref }}-windows-latest/*
+             sqld-${{ github.ref }}-macos-latest/*
+             sqld-${{ github.ref }}-ubuntu-latest/*
```

Comment:
@dyasny the release notes are already taken care of by release-drafter: https://github.com/release-drafter/release-drafter, please do no override the body 🙏

Reply:
Does the drafter place a text file somewhere? I can simply point at it

Reply:
sure, I can do that. Maybe, since there's a process in place already, it's better to merge the two pipelines instead of having them? I can add the steps wherever they are most relevant and add conditions for the release to only get generated on a 'v*.*.*' tag creation for example. I assumed there wasn't anything in place because Glauber asked me to create this pipeline

Reply:
there isn't a workflow, the releases are drafted by the bot that is configured there: https://github.com/libsql/sqld/blob/main/.github/release-drafter.yml. When we are ready to cut a release, we bump the crate version and create the release from the github interface. This will generate a tag, and create the release.